Ghidra/Processors/V850E2M/data/languages/Instructions/Divide.sinc

```diff
@@ -0,0 +1,53 @@
+##################################################
+#####				Divide  				 #####
+##################################################
+
+
+# DIV reg1, reg2, reg3 - rrrrr111111RRRRR|wwwww01011000000
+:div R0004, R1115, R2731 is op0510=0x3F & R0004 & R1115; op1626=0x2C0 & R2731
+{
+	$(OV) = ((R1115 == 0x80000000 && R0004 == 0xFFFFFFFF) || R0004 == 0x0);
+	R2731 = R1115 s% R0004;
```

Comment:
This is tricky, if 2731 == 1115 or 2731 == 0004 then the divide is corrupted.  I would save the registers into locals and then you can safely do the right hand side of modulus and div calculation with those.